source/_components/image_processing.facebox.markdown

```diff
@@ -118,6 +118,56 @@ A valid service data example:
 ```
 {% endraw %}
 
+You can use an automation to receive a notification when you train a face:
+
+{% raw %}
+```yaml
+- id: '1533703568569'
+  alias: Face taught
+  trigger:
+  - event_data:
+      service: facebox_teach_face
+    event_type: call_service
+    platform: event
+  condition: []
+  action:
+  - service: notify.pushbullet
+    data_template:
+      message: '{{ trigger.event.data.service_data.name }} taught 
+      with file {{ trigger.event.data.service_data.file_path }}'
+      title: Face taught notification
+```
+{% endraw %}
+
+Any errors on teaching will be reported in the logs. If you enable [system_log](https://www.home-assistant.io/components/system_log/) events:
+
+{% raw %}
```

Comment:
Only templates require the raw tags.

Reply:
Could you please expand on that a bit more? When to use, when not etc.
This is pretty unclear in the ['documentation documentation'](https://developers.home-assistant.io/docs/en/documentation_create_page.html) in my opinion.

Reply:
@tjorim my reading of it is that only the contents of the yaml (i.e. configuration) files should be enclosed in the `raw`, so my example service data should NOT be enclosed. Is this correct @MartinHjelmare ?

Reply:
We don't want jekyll to try to process the curly braces in jinja templates. So we use the raw tag to temporarily disable tag processing. Read more here eg:
http://ozzieliu.com/2016/04/26/writing-liquid-template-in-markdown-with-jekyll/

So to answer the question, only literal yaml that contains templates, ie curly braces, needs to be contained within raw tags.